frontend/src/Oauth/Authorize.jsx

```diff
@@ -85,6 +86,17 @@ export function Authorize({ navigate }) {
           });
           onLoginSuccess(restResponse, navigate);
         }
+        const event = `${redirect_url ? 'signup' : 'signin'}_${
```

Comment:
I can see this snippet in the function as well

```
initPosthog({ redirect_url, ...restResponse });
posthog.capture(event, {
  email,
  workspace_id: organization_id || current_organization_id,
});

```
Can you make sure what the difference is?

If I remember correctly, this was for failed logins.

Reply:
Did you check this scenario ?

Reply:
Also, we are setting something like this when redirecting.
```

 if (redirect_url) {
        localStorage.setItem('ph-sso-type', router.query.origin); //for posthog event
        window.location.href = redirect_url;
        return;
      }
```

could you please confirm this as well. please check the cloud code for the reference ?